### DISCUSSION TITLE
Windows 11 stopped booting out of the blue, launches in 'Preparing Automatic Repair' mode

### DISCUSSION THREAD

**Discussion by darshil-v** [utmapp/UTM#5061] [UTM Help (macOS)] [answered]
**Describe the issue**  
Have been using this Windows 11 VM for a while now and has been working perfectly fine. Last used it about an hour ago, boot it up now and ran into this issue. Checked other issue threads, tried `highmem=off,virtualization=on` QEMU options but it doesn't work with my "host CPU". Currently using M1 Pro MBP 14, macOS Ventura 13.2. Also tried every available option on the "Advanced Repair" page (including System Reset) but no option seems to work. Please find attached log and screenshot below. 

**Configuration**  
* UTM Version: 3.2.4 (58)
* macOS Version: 13.2 (22D49)
* Mac Chip (Intel, M1, ...): M1 Pro

**Debug log**  

Running:  -L /Applications/UTM.app/Contents/Resources/qemu -S -qmp tcp:127.0.0.1:4000,server,nowait -nodefaults -vga none -spice "unix=on,addr=/Users/darshilvala/Library/Group Containers/WDNLXAD4W8.com.utmapp.UTM/72E92709-CFEA-4AE9-8D53-CAA1D826929B.spice,disable-ticketing=on,image-compression=off,playback-compression=off,streaming-video=off,gl=off" -device virtio-ramfb -cpu host -smp cpus=8,sockets=1,cores=8,threads=1 -machine virt, -accel hvf -accel tcg,tb-size=2048 -drive if=pflash,format=raw,unit=0,file=/Applications/UTM.app/Contents/Resources/qemu/edk2-aarch64-code.fd,readonly=on -drive if=pflash,unit=1,file=/Users/darshilvala/Library/Containers/com.utmapp.UTM/Data/Documents/Windows11.utm/Images/efi_vars.fd -boot menu=on -m 8192 -device intel-hda -device hda-duplex -name Windows11 -device nec-usb-xhci,id=usb-bus -device usb-tablet,bus=usb-bus.0 -device usb-mouse,bus=usb-bus.0 -device usb-kbd,bus=usb-bus.0 -device qemu-xhci,id=usb-controller-0 -chardev spicevmc,name=usbredir,id=usbredirchardev0 -device usb-redir,chardev=usbredirchardev0,id=usbredirdev0,bus=usb-controller-0.0 -chardev spicevmc,name=usbredir,id=usbredirchardev1 -device usb-redir,chardev=usbredirchardev1,id=usbredirdev1,bus=usb-controller-0.0 -chardev spicevmc,name=usbredir,id=usbredirchardev2 -device usb-redir,chardev=usbredirchardev2,id=usbredirdev2,bus=usb-controller-0.0 -device usb-storage,drive=cdrom0,removable=true,bootindex=0,bus=usb-bus.0 -drive if=none,media=cdrom,id=cdrom0 -device nvme,drive=drive0,serial=drive0,bootindex=1 -drive if=none,media=disk,id=drive0,file=/Users/darshilvala/Library/Containers/com.utmapp.UTM/Data/Documents/Windows11.utm/Images/data.qcow2,discard=unmap,detect-zeroes=unmap -device virtio-net-pci,mac=46:03:E9:AA:FE:30,netdev=net0 -netdev vmnet-shared,id=net0 -device virtio-serial -device virtserialport,chardev=vdagent,name=com.redhat.spice.0 -chardev spicevmc,id=vdagent,debug=0,name=vdagent -device virtserialport,chardev=charchannel1,id=channel1,name=org.spice-space.webdav.0 -chardev spiceport,name=org.spice-space.webdav.0,id=charchannel1 -uuid 72E92709-CFEA-4AE9-8D53-CAA1D826929B -rtc base=localtime -device virtio-rng-pci
qemu-aarch64-softmmu: warning: Spice: playback:0 (0x11d0d6940): setsockopt failed, Operation not supported on socket
qemu-aarch64-softmmu: warning: Spice: record:0 (0x11d0d69e0): setsockopt failed, Operation not supported on socket

**Comment by darshil-v** [comment]
<img width="803" alt="Screenshot 2023-02-18 at 3 37 28 PM" src="https://user-images.githubusercontent.com/8133083/219854995-503e49eb-7d98-4baf-bfdc-b31265ffd0d2.png">

**Comment by osy** [comment, marked as answer]
Please upgrade to the latest UTM. There were some issues in v3 with drive corruption when a VM isn’t terminated properly.

**Reply by darshil-v** [reply]
Ended up updating the app and re-installing the VM and it works. 
Left the issue open in case somebody had any other solutions for future users.

Thanks!

**Comment by Hamzah023** [comment]
Same thing happened to me to run easy68k on a windows os, you just have to keep restarting UTM.
